.github/workflows/docs-deploy.yml

```diff
@@ -0,0 +1,28 @@
+name: Docs Deploy
+on: 
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Dump Github context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+          TEST: ${{ secrets.TEST_SECRET }}
+        run: |
+          echo "$GITHUB_CONTEXT"
+          if [[ "$TEST" == TEST* ]]; then
+            echo "secret is present"
+          else
+            echo "no secret"
+          fi
+      - name: Deploy Docs
+        env:
```

Comment:
I didn't understood, what we exactly this workflow is doing (the use case)?
why we compare ? and why are these `secrets`?